lucid/src/Lucid/Htmx.hs

```diff
@@ -1,184 +1,13 @@
-{-# LANGUAGE OverloadedStrings #-}
-
-module Lucid.Htmx
-  ( hxBoost_,
-    hxConfirm_,
-    hxEncoding_,
-    hxExt_,
-    hxDelete_,
-    hxDisable_,
-    hxGet_,
-    hxHeaders_,
-    hxHistoryElt_,
-    hxInclude_,
-    hxIndicator_,
-    hxParams_,
-    hxPatch_,
-    hxPost_,
-    hxPreserve_,
-    hxPrompt_,
-    hxPushUrl_,
-    hxPut_,
-    hxRequest_,
-    hxSelect_,
-    hxSse_,
-    hxSwapOob_,
-    hxSwap_,
-    hxTarget_,
-    hxTrigger_,
-    hxVals_,
-    hxWs_,
-    useHtmx,
-    useHtmxExtension,
-    useHtmxVersion,
-    useHtmxVersionExtension,
-  )
+module Lucid.Htmx (
+    module Lucid.Htmx.Core,
+    module Lucid.Htmx.Extra,
+    module Lucid.Htmx.Head,
+    module Lucid.Htmx.Event,
+)
```

Comment:
The re-exports mean that existing dependents of the library shouldn't notice any change